app/src/main/res/values/strings.xml

```diff
@@ -7,46 +7,46 @@
     <string name="dialog_metered_network_message_default">Your device is connected to a metered network.</string>
     <string name="dialog_metered_network_title_default">Metered Network</string>
     <string name="dialog_metered_update_collection_button_okay">Update anyway</string>
-    <string name="dialog_metered_update_collection_message">Your device is not connected to a Wifi network. Do you want to update your podcasts anyway?</string>
-    <string name="dialog_metered_update_collection_title">Update without Wifi</string>
+    <string name="dialog_metered_update_collection_message">Your device is not connected to Wi-Fi. Update your podcasts anyway?</string>
+    <string name="dialog_metered_update_collection_title">Update without Wi-Fi</string>
     <string name="dialog_opml_import_button_okay">Import</string>
     <string name="dialog_opml_import_details_button">Show details</string>
     <string name="dialog_opml_import_details_default">No details available</string>
-    <string name="dialog_opml_import_message_error">Unable to import OPML file. No podcasts found.</string>
-    <string name="dialog_opml_import_message">Do you want to add new podcasts? Duplicates will be removed before importing. Number of podcasts found:</string>
+    <string name="dialog_opml_import_message_error">Could not import OPML file. No podcasts found.</string>
+    <string name="dialog_opml_import_message">Add new podcasts? Duplicates will be removed before importing. Number of podcasts found:</string>
     <string name="dialog_opml_import_title">OMPL Import</string>
     <string name="dialog_show_notes_close">Close</string>
-    <string name="dialog_show_notes_title">Show Notes</string>
-    <string name="dialog_yes_no_message_add_up_next">Do you want to start listening to this episode right away?</string>
+    <string name="dialog_show_notes_title">Shownotes</string>
+    <string name="dialog_yes_no_message_add_up_next">Start listening to this episode right away?</string>
     <string name="dialog_yes_no_message_default">Do you really want to do this?</string>
-    <string name="dialog_yes_no_message_delete_downloads">Do you want to delete all downloaded episodes? You can re-download them afterwards.</string>
-    <string name="dialog_yes_no_message_delete_episode">Do you want to delete this episode?</string>
-    <string name="dialog_yes_no_message_mark_episode_played">Do you want to mark this episode as played?</string>
-    <string name="dialog_yes_no_message_remove_podcast">Do you want to remove this podcast?</string>
-    <string name="dialog_yes_no_message_update_collection">Do you want to check, if there are new episodes available?</string>
-    <string name="dialog_yes_no_negative_button_add_up_next">Add to Up Next</string>
+    <string name="dialog_yes_no_message_delete_downloads">Delete all downloaded episodes? You can re-download them afterwards.</string>
+    <string name="dialog_yes_no_message_delete_episode">Delete this episode?</string>
```

Comment:
I see what you intent to do here. I think your phrasing is more precise. But: The title of this dialog is "Delete Episode". Isn't it a bit problematic if the content of that dialog is so similar. In this case "Delete this episode?"? That was why I used the phrasing "Do you want to ..." in some of the dialog messages.
![IMG_20191002_132406](https://user-images.githubusercontent.com/9103935/66040707-71a31600-e518-11e9-9a33-dad95634f04f.jpg)

Reply:
I think it is fine for now. The greater question is why it is redundant in the first place? It could be just "Delete?" or "Delete _episode name_? Also the "OK" is not an OK.
Why is is this whole ordeal a pop-up? How about a "Delete" right next to it, and then the three second animation before it actually happens?

Reply:
True. I guess I should get rid of the redundancy. I checked out the [interactive dialog demo](https://material.io/components/dialogs/) on the Material Design site. Turns out the dialog title is optional.